src/ORM/TableRegistry.php

```diff
@@ -184,7 +184,7 @@ public static function get($alias, array $options = [])
         }
 
         if (isset(static::$_config[$alias])) {
-            $options += static::$_config[$alias];
+            $options = static::$_config[$alias] + $options;
```

Comment:
Hmm line 168 in existing code already has `static::$_options[$alias] = $options;` so i wonder if this merge is required at all.